azure-webapp-maven-plugin/CHANGELOG.md

```diff
@@ -16,6 +17,12 @@ All notable changes to the "Maven Plugin for Azure App Service" will be document
   - [1.1.0](#110)
   - [1.0.0](#100)
 
+## 1.9.0
+- Support deploy external resources to Azure outside `wwwroot`.[PR#953](https://github.com/microsoft/azure-maven-plugins/pull/953)
+- Support specify authentication method in configuration with parameter `<authType>`.[#PR#975](https://github.com/microsoft/azure-maven-plugins/pull/975)
+- Validate subscription in authentication and prompt more friendly error messages.[#931](https://github.com/microsoft/azure-maven-plugins/issues/931)
```

Comment:
Validate subscription in authentication and prompt more friendly error messages -> Validate null subscription id and provide friendly error messages when authentication fails

Reply:
Done.